azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/legacy/function/handlers/CommandHandlerImpl.java

```diff
@@ -62,7 +64,7 @@ protected String getOutputFromProcess(final Process process) throws IOException
     protected Process runCommand(final String command,
                                  final boolean showStdout,
                                  final String workingDirectory) throws IOException, InterruptedException {
-        Log.debug("Executing command: " + StringUtils.join(command, " "));
+        log.debug("Executing command: " + StringUtils.join(command, " "));
```

Comment:
replace with `AzureMessager.getMessager().log`?

Reply:
debug message should not show to user. so debug should not be part of AzureMessager.